sympy/combinatorics/perm_groups.py

```diff
@@ -1895,7 +1895,7 @@ def is_primitive(self, randomized=True):
     def minimal_blocks(self, randomized=True):
         '''
         For a transitive group, return the list of all minimal
-        block systems. If a group is intrasitive, return `False`.
+        block systems. If a group is intransitive, return `False`.
```

Comment:
I'm not sure if the term 'intransitive' is used or 'non transitive' is used.

Reply:
'intransitive' seems to be used in some books on group theory.

Reply:
Agreed. Its even used in the Handbook that way.

Reply:
I think 'intransitive' is right.